### PR TITLE
Merge release 1.2.0 into develop 

### PR DESCRIPTION
**Pull request by jcmincke**
Release/1.2.0 -> develop: final merge.

### CODE REVIEW

File: release.nix [hunk 1]
```diff
@@ -46,7 +46,11 @@ let
   swaggerSchemaValidation = iohkPkgs.swaggerSchemaValidation;
   walletIntegrationTests = iohkPkgs.buildWalletIntegrationTests;
 in (mapTestOn platforms) // {
+<<<<<<< HEAD
   inherit stagingWalletdockerImage walletIntegrationTests swaggerSchemaValidation shellcheckTests;
+=======
```
Comment (adinapoli-iohk) [inline release.nix]: @jcmincke @rvl Ops!

File: wallet-new/README.md [hunk 2]
```diff
@@ -73,13 +73,35 @@ $ stack exec cardano-node -- --topology=wallet-new/topology-examples/testnet.yam
 
 From there, you can browse the API documentation for V0 and V1 through the following URLs:
 
+<<<<<<< HEAD
```
Comment (adinapoli-iohk) [inline wallet-new/README.md]: @jcmincke Argh, he it goes another conflict.

I would use the version of the readme shipped with `1.2.0` (so let's remove here everything from <<<HEAD to ====)

File: wallet-new/README.md [hunk 3]
```diff
@@ -200,6 +222,7 @@ using environment variables as follows:
 ```
 LANG=en_GB.UTF-8 LC_ALL=en_GB.UTF-8 stack exec -- ...
 ```
+<<<<<<< HEAD
```
Comment (adinapoli-iohk) [inline wallet-new/README.md]: This is useful to include instead, we should maintain in my opinion this section about "API returns 415 unsupported ..."

File: wallet-new/src/Cardano/Wallet/API/V1/Types.hs [hunk 4]
```diff
@@ -796,7 +796,11 @@ instance BuildableSafeGen Wallet where
     %" balance="%buildSafe sl
     %" }")
     walId
-    walName
+    -- TODO(parsons.matt): remove this when we find out *why* this is
+    -- causing it to fail.
+    --
+    -- https://iohk.myjetbrains.com/youtrack/issue/R120-4#comment=93-20855
+    (T.filter C.isAscii walName)
```
Comment (adinapoli-iohk) [inline wallet-new/src/Cardano/Wallet/API/V1/Types.hs]: @parsonsmatt @KtorZ Would you guys refresh me on whether or not on `develop` we eventually fixed this properly on the `log-warper` side?

If yes, @jcmincke could remove this "hotfix" (i.e. get rid completely of this patch, and revert to simply `walName`).

Comment (KtorZ) [reply wallet-new/src/Cardano/Wallet/API/V1/Types.hs]: We fixed the log-warper on develop, actually using a different (patched) tip for the package.
See https://github.com/input-output-hk/cardano-sl/pull/2843

Nevertheless, develop _also_ contains the hotfix setting the `LANG` & `LC_ALL` env vars. In theory, it shouldn't matter. Yet I am not sure we want to remove it at this point.

**Pull request merged**